pysoa/common/transport/base.py

```diff
@@ -12,9 +12,15 @@
 types, the metadata may include a mime type to tell the endpoint receiving the message
 which type of serializer to use.
 """
+import threading
+
 from pysoa.common.metrics import NoOpMetricsRecorder
 
 
+def get_hex_thread_id():
+    return '{:012x}'.format(hash(threading.current_thread().ident))
```

Comment:
Minor nitpick:

I think `hash()` can be skipped. `.ident` is already guaranteed to be a unique integer. 

With high enough integers `hash` can cause clashes,
for example `hash(2305843009213693952) == hash(1)`
The clash is highly improbable, but all it saves is 4 characters. 
So why not `'{:016x}'.format(threading.current_thread().ident)`?

Reply:
Good point. Done.